src/System.IO.Compression.ZipFile/src/System/IO/Compression/ZipFile.cs

```diff
@@ -8,6 +8,9 @@ namespace System.IO.Compression
 {
     public static class ZipFile
     {
+        // Per the .ZIP File Format Specification 4.4.17.1 all slashes should be forward slashes
+        private const char c_pathSeparator = '/';
```

Comment:
Up to you.  I'm fine either fixing it up in TFS or letting this go through and addressing it with a separate PR on GitHub.  Not a big deal.